test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/OperationVersionsTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Health.Fhir.Tests.Common.FixtureParameters;
+using Microsoft.Health.Fhir.Tests.E2E.Rest;
+using Microsoft.Health.Fhir.Web;
+using Microsoft.Net.Http.Headers;
+using Xunit;
+
+namespace Microsoft.Health.Fhir.Shared.Tests.E2E.Rest
+{
+    [HttpIntegrationFixtureArgumentSets(DataStore.CosmosDb, Format.Json)]
+    public class OperationVersionsTests : IClassFixture<HttpIntegrationTestFixture<Startup>>
+    {
+        private readonly HttpClient _client;
+
+        public OperationVersionsTests(HttpIntegrationTestFixture<Startup> fixture)
+        {
+            _client = fixture.HttpClient;
+        }
+
+        [Theory]
+        [InlineData("application/json")]
+        [InlineData("application/xml")]
+        [InlineData("application/fhir+json")]
+        [InlineData("application/fhir+xml")]
+        [InlineData("*/*")]
+        public async Task WhenVersionsEndpointIsCalled_GivenValidAcceptHeaderIsProvided_ThenServerShouldReturnOK(string acceptHeaderValue)
+        {
+            HttpRequestMessage request = GenerateOperationVersionsRequest(acceptHeaderValue);
+            HttpResponseMessage response = await _client.SendAsync(request);
+
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
```

Comment:
Can we validate `response.Content.Headers.ContentType` to see what was returned?

Reply:
Yep makes sense. Let me know what you think of c2f9855.

Reply:
looks good